<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Match</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="showTemplate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>271</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color:rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="selectTemplate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择模板</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seletSrc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择原图</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thresh">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0.73</string>
    </property>
   </widget>
   <widget class="QLabel" name="threshLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>阈值</string>
    </property>
   </widget>
   <widget class="QLabel" name="showSrc">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color:rgb(255,255,255);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="minContrastLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>下限</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minContrast">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>66.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxContratLabl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>上限</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxContrast">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>100.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pyramidLayerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>金字塔层数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pyramidLayer">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="greedinessLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>贪心值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="greediness">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0.8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startMatch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始匹配</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveResult">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>结果保存</string>
    </property>
   </widget>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color:rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>模板图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>原图像</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
